templates/tab/js/m365/package.json

```diff
@@ -8,8 +8,9 @@
     "@microsoft/mgt-react": "2.4.1-next.teamsfx.8d56785",
     "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.8d56785",
     "@microsoft/microsoft-graph-client": "^3.0.1",
-    "@microsoft/teams-js": "^2.0.0",
-    "@microsoft/teamsfx": "^2.0.0-beta.0",
+    "@microsoft/teams-js": "^2.2.0",
+    "@microsoft/teamsfx": "2.0.0-beta.0",
+    "@microsoft/teamsfx-react": "2.0.0-beta.1",
```

Comment:
Fix to this version?

Reply:
Yes, ^ will automatically install 2.0.0-experimental.0, which is not what we would like to see.